.travis.yml

```diff
@@ -1,15 +1,28 @@
-sudo: false
-
 language: python
-
+# dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
 python:
-  - 2.7
-
+    - 2.7
+    - 3.6
+    - 3.7
 env:
-  - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
-  - TOXENV=django111
+    - TOXENV=django111
+    - TOXENV=django20
+    - TOXENV=django21
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=quality
+    - python: 3.7
+      env: TOXENV=quality
+  exclude:
+    - python: 2.7
+      env: TOXENV=django20
+    - python: 2.7
+      env: TOXENV=django21
+  allow_failures:
+    - python: 3.6
+    - python: 3.7
+    - env: TOXENV=quality
```

Comment:
Hmm, we don't want to leave the quality check failing for Python 2.7.  It looks like the failures are due to new [checks introduced in pylint-django 0.11](https://github.com/PyCQA/pylint-django/blob/master/CHANGELOG.rst#version-011-18-april-2018-the-testcon-moscow-edition); could you take a stab at fixing those?  It looks like it should only require minor changes in `search/views.py`.

Reply:
Fixed.